src/lib/Http/Session/Flash/NotificationBag.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Http\Session\Flash;
+
+use EzSystems\HybridPlatformUi\Notification\Notification;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Used for adding notifications.
+ */
+class NotificationBag
+{
+    const FLASH_MESSAGE_TYPE = 'hyybrid-notification';
```

Comment:
@dpobel did we not say that we wanted to have notifications per element ? Like tabs notifications ?